.github/workflows/actions.yml

```diff
@@ -0,0 +1,56 @@
+name: install-and-test-workflow
```

Comment:
what is this doing?

Reply:
Ok will comment until there are tests

Reply:
https://github.com/nomad-hzb/hzb-combinatorial-libraries-plugin/blob/6-add-actions-gitignore-and-docs/tests/test_schema.py there's one test